apiary.apib

```diff
@@ -10269,28 +10276,38 @@ Displays the hashes of all local objects.
 
 + Response 200
 
+    The response has been truncated, to show only a sample of identically-formatted ndjson objects.
+
     + Headers
 
         ```
         Access-Control-Allow-Headers: X-Stream-Output, X-Chunked-Output, X-Content-Length
         Access-Control-Expose-Headers: X-Stream-Output, X-Chunked-Output, X-Content-Length
-        Content-Type: text/plain
-        Server: go-ipfs/0.4.1-dev
+        Content-Type: application/json
+        Server: go-ipfs/0.4.2
         Trailer: X-Stream-Error
-        X-Stream-Output: 1
-        Date: Tue, 19 Apr 2016 14:11:14 GMT
+        Vary: Origin
+        X-Chunked-Output: 1
+        Date: Mon, 13 Jun 2016 10:58:20 GMT
         Transfer-Encoding: chunked
         ```
 
-    + Attributes (string)
+    + Attributes (ndjson)
 
-        Newline delimited multihashes
+        - Ref (Multihash)
+        - Err (string)
 
     + Body
 
         ```
-        QmNLjhs2cWVGopVTE5KqCZBdX7TE6i9AU1y8pwytt7cyzC
-        QmNLwdgyRiTuMf9M3Fjtu2TpMjz2CgZS4YgeuqjLHv3rjq
+        {
+          "Ref": "QmNLxtKFa66yoRQHWa4FXkwCMqVpWGhg2XBXAfwKvSmJFp",
+          "Err": ""
+        }
+        {
+          "Ref": "QmNXFHU3KzhzXN9pZ5Bdghk3LYGj58Xzi7WvMVVoykdLxA",
+          "Err": ""
+        }
```

Comment:
I automatically format it to make it more readable, as all of the JSON is basically scrunched up in the terminal.

I could add a note about this to make this clear?